# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/ScaledFontMac.cpp
# Commit: e5570884e8e4
# Full Hash: e5570884e8e43505e1a35ce3fa7ab8fc33ede79d
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-04-16 09:48:54
# Regressor Bug: 1821512
# File Overlap Count: 1
# Description:
#   Bug 1821512 - Fix Skia usage for API changes. r=jrmuizel
#   
#   This just tries to address fairly random changes in the Skia API and correct
#   our usage of it in Moz2D and some other places.
#   
# ==============================================================================

diff -r efc92c86fafd -r e5570884e8e4 gfx/2d/ScaledFontMac.cpp
--- a/gfx/2d/ScaledFontMac.cpp	Sun Apr 16 03:59:59 2023 +0000
+++ b/gfx/2d/ScaledFontMac.cpp	Sun Apr 16 04:00:00 2023 +0000
@@ -164,7 +164,7 @@
 }
 
 SkTypeface* ScaledFontMac::CreateSkTypeface() {
-  return SkCreateTypefaceFromCTFont(mCTFont);
+  return SkMakeTypefaceFromCTFont(mCTFont).release();
 }
 
 void ScaledFontMac::SetupSkFontDrawOptions(SkFont& aFont) {